aospy_synthetic/test/test_db.py

```diff
@@ -3,78 +3,98 @@
 import unittest
 from subprocess import call
 
-from test_objs import (cases, models, example_projects, variables,
-                       regions)
-from aospy_synthetic.calc import Calc, CalcInterface
+from test_objs import (runs, models, projects, variables,
+                       regions, calc_objs)
 from aospy_synthetic.db.sqlalchemy.sqlalchemy_db import SQLAlchemyDB
-#from aospy_synthetic.SQLAlchemyDB import SQLBackend
+from aospy_synthetic.db.sqlalchemy.sqlalchemy_config import (ProjDB, ModelDB,
+                                                             RunDB, VarDB,
+                                                             RegionDB, CalcDB)
 
 
-class AospySyntheticTestCase(unittest.TestCase):
+class TestProjDB(unittest.TestCase):
     def setUp(self):
-        self.proj = example_projects.example
-        self.model = models.am2
-        self.run = cases.am2_control
-        self.variable = variables.mse
-        self.variable2 = variables.dse
+        self.db = SQLAlchemyDB()
+        self.AospyObj = projects.p
+        self.db_cls = ProjDB
+        self.ex_str_attr = 'direc_out'
 
     def tearDown(self):
```

Comment:
Yep, the context manager should take care of closing the active sessions, so tearDown (in this test class) only needs to remove the database.